lib/rules/object-shorthand.js

```diff
@@ -65,8 +65,13 @@ module.exports = function(context) {
                 return;
             }
 
+            // only computed methods can fail the following checks
+            if (!APPLY_TO_METHODS && node.computed) {
+                return;
+            }
+
             // getters, setters and computed properties are ignored
```

Comment:
this comment is no longer true i think? "and computed properties" should be removed